schemas/display-mapping-object.json

```diff
@@ -24,19 +24,41 @@
               "required": ["type"]
             },
             {
-              "type": "object",
-              "additionalProperties": false,
-              "properties": {
-                "type": {
-                  "type": "string",
-                  "enum": ["string"]
+              "anyOf": [
```

Comment:
This schema works but I'm wondering if there's a better way of defining this.

This is how I'd define the types in TS

```ts
type StringFormatDMO = {
  type: 'string'
  format?: 'date-time' | 'time' | 'date' | 'etc.'
  contentEncoding?: never
  contentMediaType?: never
}

type StringEncodingDMO = {
  type: 'string'
  format?: never
  contentEncoding?: '7bit' | '8bit' | 'binary' | 'etc.'
  contentMediaType?: string
}

type StringDMO = StringFormatDMO | StringEncodingDMO

type OtherDMO = {
  type: 'boolean' | 'number' | 'integer'
}

type DMO = StringDMO | OtherDMO
```